modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+The following premium plugin updates were released alongside {productname} 6.1.
+
+=== A11ychecker 3.0.2
+
+The {productname} 6.1 release includes an accompanying release of the **A11ychecker** premium plugin.
+
+**A11ychecker** 3.0.2 fixes the following UI bug:
+
+The *Tools > Accessibility Checker* command opens a dialog. As such, its menu title should be *Accessibility Checker…*.
+
+The trailing ellipsis indicates the command will not execute without further user input and is required for all menu commands that present dialogs.
+
+This update corrects both the menu title and the equivalently mis-presented toolbar tool tip text. Both strings now include the required ellipsis and present as *Accessibility Checker…*, as expected.
+
+For information on the **A11ychecker** plugin, see: xref:a11ychecker.adoc[A11ychecker plugin].
+
+
+=== Comments 3.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.1.0 introduces the following enhancements:
+
+
+==== A new event log API
+
+A new API, `tinymce.activeEditor.plugins.tinycomments.getEventLog()` is now available.
+
+As its name suggests, it returns a log of all Comment-related events, including:
+
+* when a new comment is added (and who added it).
+
+* when a coment is edited (and who edited it).
+
+* when a reply to a comment is added (and who added the reply).
+
+* when a comment is resolved (and who resolved the comment).
+
+* when a comment is deleted (and who deleted the comment).
+
+The new API also supports an `after` option.
+
+[source,js]
+////
+tinymce.activeEditor.plugins.tinycomments.getEventLog(
+	{after: '<iso-8601-format-time-stamp>'}
+	)
+////
+
+restricts the returned list to Comment events after the time-stamp date.
+
+Underlying this new API is a new `CommentChange` event, which gets fired when any modification is made to Comments data.
+
+
+==== Avatars can now be displayed next to usernames in comments
+
+The default icon presented next to user names in comments can now be replaced by a custom avatar.
+
+In embedded mode, the avatar is supplied by the `tinycomments_author_avatar` option:
+
+[source,js]
+----
+    plugins: [
+        "tinycomments",
+    ],
+	tinycomments_mode: "embedded",
+	tinycomments_author: "testuser",
+	tinycomments_author_name: "Test User",
+	tinycomments_author_avatar: "https://example.com/image.ext",
+----
+
+In callback mode, the avatar is supplied by the `avatar` option of `setConverstation()`:
+
+[source,js]
+----
+const avatarUrl = "https://example.com/image.ext";
+
+/*
+
+much non-avatar-related configuration here
+
+*/
+
+      setConversation(conversationUid, [
+        {
+          author,
+		  avatar: avatarUrl,
+          createdAt,
+          modifiedAt: createdAt,
+          content,
+          uid: conversationUid /* first comment has same uid as conversation */,
+        },
+      ]);
+----
+
+In both modes the avatar:
+
+* must be 36px by 36px.
```

Comment:
This isn't correct... the image can be any size and we have the CSS configured to scale it. For best presentation though, it should be something that will fit inside a circle with a 36px diameter.